modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-configuring-storage-class-bootsource-update_{context}"]
+= Configuring a storage class for custom boot source updates
+
+You must configure a storage class that allows automatic updating for custom boot sources.
+
+.Procedure
+
+* Edit the HyperConverged custom resource (CR) and define a `storageClassName`:
++
+[source,yaml]
+----
+...
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  dataImportCronTemplates:
+  - metadata:
+      name: rhel8-image-cron
+    spec:
+      template:
+        spec:
+          storageClassName: "some-name"
```

Comment:
If the value of this field is to be added by the user, format it as a variable. Maybe something like `<storage_class_name>`

Reply:
Done.